birdhouse/components/weaver/config/weaver/weaver.ini.template

```diff
@@ -85,6 +85,10 @@ weaver.wps_email_notify_ssl = true
 weaver.wps_email_notify_template_dir =
 weaver.wps_email_notify_template_default =
 
+# --- Weaver vault configuration ---
+weaver.vault = true
+weaver.vault_dir = /tmp/vault
```

Comment:
The default `weaver.vault_dir` is under `/tmp`?

Shouldn't it be somewhere else that is persisted to disk?  Just my 2 cents. I do not really know what this does.

Reply:
No. In it in fact better to not have it persisted because it is intended for single use and forget situations to transfer secrets or other sensible data as input for some process execution.